addons/mail/static/src/models/rtc_session/rtc_session.js

```diff
@@ -248,6 +249,21 @@ registerModel({
                 f();
             }, delay);
         },
+        /**
+         * Ensures that we do not download the stream of videos that don't have views.
+         *
+         * @private
+         */
+        _onChangeCallParticipantCards() {
+            if (!this.channel || !this.channel.rtc) {
+                return;
+            }
+            if (this.callParticipantCards && this.callParticipantCards.length === 0) {
+                this.channel.rtc.disallowVideoReceiverActivity(this.id);
+            } else {
+                this.channel.rtc.allowVideoReceiverActivity(this.id);
```

Comment:
Pass `RtcSession` instead of `RtcSession/id`.
It's easier to deal with records than record fields.

Reply:
Many attributes in `Rtc` should become actually `RtcSession` fields. This includes `_peerConnections[token]`.
The indirection here is caused by bad split.

It's not a big deal in this PR, but just saying that this `onChange` could be much smaller (1-liner `onChange`) and focus precisely on technical aspect of changing
```
{RtcSession/web.rtcPeerConnection}
.{web.RTCPeerConnection/videoTransceiver}
.{web.RTCRtpTransceiver/direction}
```